addon/components/img-lazy.js

```diff
@@ -65,6 +65,14 @@ export default Component.extend({
     get(this, 'observer').observe(this)
   },
 
+  didReceiveAttrs() {
+    this._super(...arguments);
+
+    if (this._src !== this.getAttr('src')) {
```

Comment:
Where is this `getAttr` method coming from? I don't see anything about this in the official ember docs. :thinking:

Reply:
At some point Ember core members had recommended using `getAttrs` over `this.attrs`, but sadly I still don't think it's documented outside of test coverage. https://github.com/emberjs/ember.js/blob/d88da1b61d8dfc15138958682252c3bd1f2a28fb/packages/%40ember/-internals/glimmer/tests/integration/components/attrs-lookup-test.js#L141

Reply:
Could you do this instead?
```
if (this._src !== this.src) {
```

Reply:
Yup, that works too. I've pushed a new commit with that change.  @topaxi any chance you could take another look at this? Thanks in advance!

Reply:
@topaxi Any chance we can take at this one again?